### PR TITLE
Adding courses flow

### PR DESCRIPTION
**Pull request by PeterChu2**
can add courses via. the + button on the bottom right now. Once the schedules db works, we should make the user add course code + input for section number.

implemented by saving a string set of course ids in the user's shared preferences, and loading them on opening the app

### CODE REVIEW

**Comment by PeterChu2** [comment]
matches course code input of lower case, upper case, with/without space. Will show the user an error if the course is not offered for the term

**Comment by timothytong** [comment]
whoops sorry4conflicts

File: app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java [hunk 1]
```diff
@@ -213,6 +213,43 @@ public Course getCourse(int id) {
         return course;
     }
 
+    // Get courses by list of ids
+    public List<Course> getCourses(String[] ids) {
```
Comment (timothytong) [inline app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java]: why would we want to get courses by id's? Its not like we have a list of id's readily available, or do we?

Comment (PeterChu2) [inline app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java:217]: i save it in sharedpreferences when the user adds a course. So we can fetch all the courses in the user's sharedpreferences at once

Comment (timothytong) [inline app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java]: ok i see how its used now

**Review by timothytong — APPROVED** [review]
LGTM

**Pull request merged**